LayoutTests/fast/forms/form-control-refresh/autofilled-and-obscured-text-input-expected.html

```diff
@@ -0,0 +1,26 @@
+<html>
+<head>
+    <script>
+        function setBackground() {
+            var tf = document.getElementById('devolved');
```

Comment:
Nit: Here and elsewhere, we should use full words: https://webkit.org/code-style-guidelines/#names-full-words